style.css

```diff
@@ -0,0 +1,69 @@
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #2b57a8;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
```

Comment:
- [ ] Kindly make sure your header looks like the figma design as much as possible, (font-family ,font-weight,text-decoration, humburger button)

  - Your design:
![nav](https://user-images.githubusercontent.com/97834160/205599737-c3a6e85c-8418-494f-990d-d41cff9efb55.png)


  - Figma:
![navFig](https://user-images.githubusercontent.com/97834160/205599765-b5244b4c-620d-4290-aee5-8a2076472536.png)